File: 1217. Play with Chips/_1217.py
class Solution:
    def minCostToMoveChips(self, chips) -> int:
        ans = [0,0]
        for i in chips:
            ans[i%2] += 1
        return min(ans)

    def minCostToMoveChips1(self, chips) -> int:
        odds = sum(x % 2 for x in chips)
        return min(odds, len(chips) - odds)

File: 1217. Play with Chips/1217.go
package leetcode

func minCostToMoveChips(chips []int) int {
	ans := make([]int, 2)
	for _, v := range chips{
		ans[v%2] ++
	}
	if ans[0]<ans[1]{
		return ans[0]
	}
	return ans[1]
}